src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -1130,19 +1147,49 @@ subroutine atm_srk3(domain, dt, itimestep, exchange_halo_group)
             call mpas_timer_start('atm_compute_dyn_tend')
    
             allocate(delsq_theta(nVertLevels,nCells+1))
-            delsq_theta(:,nCells+1) = 0.0_RKIND
+            !$acc parallel default(present)
+            !$acc loop vector
+            do k = 1, nVertLevels
+            delsq_theta(k,nCells+1) = 0.0_RKIND
```

Comment:
Here and in several of the loops below, could we indent the loop body?

Reply:
thanks for the catch!